### PR TITLE
timgm6mb-soundfont: Add new package

### PR DESCRIPTION
**Pull request by hliebel**
This is an older file, but is small and seems to be kept around by people like Debian. Alternatives are much bigger.

### CODE REVIEW

**Review by alarcher — COMMENTED** [review]
Does it work if you gmake publish several times without clobber?

**Comment by hliebel** [comment]
Hi.
What do you mean with 'work', the build scripts?
If I do:
1) gmake clobber
2) gmake publish -> PUBLISHED
2) gmake publish -> gmake: Nothing to be done for 'publish'.
3) touch Makefile
4) gmake publish -> PUBLISHED

Thanks,
Harry

File: components/multimedia/soundfont/timgm6mb/Makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+COMPONENT_LICENSE_FILE=	timgm6mb-soundfont.license
+COMPONENT_LICENSE=	GPLv2
+
+CONFIGURE_PREFIX=/usr
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+SF2_SHARE_DIR=$(CONFIGURE_PREFIX)/share/sounds/sf2
+PKG_OPTIONS +=  -D SF2_SHARE_DIR="$(SF2_SHARE_DIR)"
```
Comment (alarcher) [inline components/multimedia/soundfont/timgm6mb/Makefile]: You should be able to append to PKG_MACROS directly.

File: components/multimedia/soundfont/timgm6mb/timgm6mb-soundfont.p5m [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=pkg.description value="$(COMPONENT_DESCRIPTION)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+dir path=$(SF2_SHARE_DIR)/
```
Comment (pyhalov) [inline components/multimedia/soundfont/timgm6mb/timgm6mb-soundfont.p5m]: don't include last "/" in dir action. Also this dir action is not necessary, as we have file action

File: components/multimedia/soundfont/timgm6mb/Makefile [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#
```
Comment (pyhalov) [inline components/multimedia/soundfont/timgm6mb/Makefile]: Do we really have to place it in 'components/multimedia/soundfont/timegm6mb'? I'd put it one level upper (avoid creating soundfont directory to hold one component).

File: components/multimedia/soundfont/timgm6mb/Makefile [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+PKG_OPTIONS += -D SF2_SHARE_DIR="/usr/share/sounds/sf2"
+
+# common targets
+build:		prep
+
+install:	build
+
```
Comment (pyhalov) [inline components/multimedia/soundfont/timgm6mb/Makefile]: gmake clean doesn't remove build directory.
Consider adding clean:: action

**Review by alarcher — APPROVED** [review]

**Pull request merged**